File: ChuckNorrisIO/app/src/main/java/br/com/titoschmidt/chucknorrisio/models/CategoryItem.java
package br.com.titoschmidt.chucknorrisio.models;

import android.support.annotation.NonNull;
import android.widget.TextView;
import com.xwray.groupie.Item;
import com.xwray.groupie.ViewHolder;
import br.com.titoschmidt.chucknorrisio.R;

public class CategoryItem extends Item<ViewHolder> {

    private final String nomeCategoria;
    private  final int bgColor;

    public CategoryItem(String nomeCategoria, int bgColor) {
        this.nomeCategoria = nomeCategoria;
        this.bgColor = bgColor;
    }

    public String getNomeCategoria() {
        return nomeCategoria;
    }

    @Override
    public void bind(@NonNull ViewHolder viewHolder, int position) {
        TextView txtCategory = viewHolder.itemView.findViewById(R.id.txt_category);
        txtCategory.setText(nomeCategoria);
        viewHolder.itemView.setBackgroundColor(bgColor);
    }

    @Override
    public int getLayout() {
        return R.layout.card_category;
    }
}

File: JokerApp/app/src/main/java/co/titoschmidt/tutorial/jokerappdev/view/CategoryItem.kt
package co.titoschmidt.tutorial.jokerappdev.view

import android.view.View
import android.widget.LinearLayout
import android.widget.TextView
import co.titoschmidt.tutorial.jokerappdev.model.Category
import com.xwray.groupie.GroupieViewHolder
import com.xwray.groupie.Item
import titoschmidt.tutorial.jokerappdev.R

/*
 * 08/12/2021
 * @author titoesa1988@gmail.com (Tito Schmidt).
 * 
 */
class CategoryItem(val category: Category) : Item<CategoryItem.CategoryViewHolder>(){

    class CategoryViewHolder(view: View) : GroupieViewHolder(view)
    // itemView é o LinearLayout em item_category.xml
    override fun createViewHolder(itemView: View) = CategoryViewHolder(itemView)

    override fun bind(viewHolder: CategoryViewHolder, position: Int) {
        viewHolder.itemView.findViewById<TextView>(R.id.txt_category).text = category.name
        viewHolder.itemView.findViewById<LinearLayout>(R.id.container_category).setBackgroundColor(category.bgColor.toInt())
    }

    override fun getLayout(): Int {
        return R.layout.item_category
    }
}